core/lib/zksync_core/src/api_server/web3/pubsub.rs

```diff
@@ -213,23 +213,23 @@ impl PubSubNotifier<(typed::Sink<PubSubResult>, PubSubFilter)> {
 }
 
 /// Subscription support for Web3 APIs.
-#[derive(Debug, Clone)]
 pub(super) struct EthSubscribe {
-    // `jsonrpc` backend executes task subscription on a separate thread that has no tokio context.
-    pub runtime_handle: tokio::runtime::Handle,
-    active_block_subs: SubscriptionMap<typed::Sink<PubSubResult>>,
-    active_tx_subs: SubscriptionMap<typed::Sink<PubSubResult>>,
-    active_log_subs: SubscriptionMap<(typed::Sink<PubSubResult>, PubSubFilter)>,
+    blocks: broadcast::Sender<BlockHeader>,
+    transactions: broadcast::Sender<H256>,
+    logs: broadcast::Sender<Log>,
     events_sender: Option<mpsc::UnboundedSender<PubSubEvent>>,
 }
 
 impl EthSubscribe {
-    pub fn new(runtime_handle: tokio::runtime::Handle) -> Self {
+    pub fn new() -> Self {
+        let (blocks, _rx_blocks) = broadcast::channel(BROADCAST_CHANNEL_CAPACITY);
```

Comment:
Nit: You can use `_` vars here. Using `_something` implies that it's important that the variable isn't *immediately* dropped (unlike `_`, `_something` vars live until the end of the scope), which doesn't seem to be the case here.

Reply:
done